components/molecules/FeaturedHighlightsPanel/featured-highlights-panel.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+import React, { useEffect, useState } from "react";
+
+interface FeaturedHighlightsPanelProps {
+  highlightIds: { id: number; title: string }[];
+}
+const FeaturedHighlightsPanel = ({ highlightIds }: FeaturedHighlightsPanelProps) => {
+  const [host, setHost] = useState("");
+
+  useEffect(() => {
+    if (window !== undefined) {
+      setHost(window.location.origin as string);
+    }
+  }, []);
+
+  return (
+    <div className="border max-w-[18.6rem] rounded-lg p-5 space-y-4">
```

Comment:
```suggestion
    <div className="border max-w-72 rounded-lg p-5 space-y-4">
```
Please could you remove `max-w-[18.6rem]`??? it is not a good practice to hard coded 
w-72 === 18rem